edit-post/components/layout/style.scss

```diff
@@ -86,7 +86,14 @@
 	-webkit-overflow-scrolling: touch;
 
 	.edit-post-visual-editor {
-		flex-basis: 100%;
+		flex-grow: 1;
+
+		// In IE11 flex-basis: 100% cause a bug where the metaboxes area overlap with the content area.
+		// Resetting the `auto` makes it behave properly.
+		flex-basis: auto;
+		@supports (position: sticky) {
```

Comment:
Do we need this at all? In brief testing, it behaves the same in Chrome, and it would be nice if we didn't have browser-targeted styles (prone to bugginess, maintenance headaches).

Reply:
No actually we do need it. As I said, in chrome, if you have a "short" content, you'd be able to click the blank area under the content to type, if you drop it, you can't.

Reply:
Also, `flex-basis: auto;` should be the default value anyways. We should only need `flex-grow: 1` from my brief testing.

Reply:
>As I said, in chrome, if you have a "short" content, you'd be able to click the blank area under the content to type, if you drop it, you can't.

Where was this said?

Reply:
Yep true, I just tried and updated the PR

Reply:
> If the content of your post is short, you should be able to click the blank area between the metaboxes and the content to type

:) (description of the PR and also)

>  it takes the content's height which break the "click anywhere in the blank area to select the last paragraph" in these browsers).

Reply:
But yeah it wasn't clear in the code, I tweaked the comment